### PR TITLE
feat(GraphQL): RHICOMPL-2038 expose tags on systems

### PR DESCRIPTION
**Pull request by skateman**
@bastilian FYI ths is how you can do it from the frontend:
```
      {
        systems {
          edges {
            node {
              id
              name
              tags {
                namespace
                key
                value
              }
            }
          }
        }
      }
```

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: app/graphql/schema.graphql [hunk 1]
```diff
@@ -545,6 +546,17 @@ type SystemEdge {
   node: System
 }
 
+"""
+A host inventory tag
+"""
+type Tag implements Node {
+  globalId: ID!
```
Comment (vkrizan) [inline app/graphql/schema.graphql:553]: Where/how is the `globalId` provided?

Comment (skateman) [reply app/graphql/schema.graphql:553]: It's been auto generated :shrug: do you know how can I get rid of it? I guess some magic declaration in the type def :thinking:

Comment (skateman) [reply app/graphql/schema.graphql:553]: Okay, I think I have it...will figure this out

**Review by vkrizan — APPROVED** [review]
LGTM

**Pull request merged**